### PR TITLE
Give the ability to set "useNativeFunction" for the reader

### PR DESCRIPTION
**Pull request by ssm2017**
hello
i have a special ini to read.
the native function can not read it (except with raw values).
the $useNativeFunction property is private so here is a patch to be able to set it from the outside.

### CODE REVIEW

File: src/IniWriter.php [hunk 1]
```diff
@@ -110,7 +110,7 @@ public function writeToString(array $config, $header = '')
     private function encodeValue($value)
     {
         if (is_bool($value)) {
-            return (int) $value;
+            return 'true';
```
Comment (tsteur) [inline src/IniWriter.php]: @ssm2017 sorry for the extremely late  reply on this. Do you maybe remember why this was changed? I think we would like to keep `(int) $value`

Comment (tsteur) [reply src/IniWriter.php]: @ssm2017 be good if you could let us know whether you're still working on this PR or not.

Comment (ssm2017) [reply src/IniWriter.php]: hello
i do not use this code anymore.
i was using your library for a custom php code.
i do not remember why i have submitted this pr.

Comment (tsteur) [reply src/IniWriter.php]: Cheers for letting us know 👍

**Pull request closed without merging**